spec/rails_helper.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../config/environment', __dir__)
+
+abort("The Rails environment is running in production mode!") if Rails.env.production?
+require 'rspec/rails'
+begin
```

Comment:
you could add a newline before this `begin` and before `RSpec.configure`, it will make blocks easier to read

Reply:
This file was auto-generated by rspec, but I will do that!